### PR TITLE
Modify S3281(xml): Migrate to LayC

### PR DESCRIPTION
**Pull request by loris-s-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S3281/xml/rule.adoc [hunk 1]
```diff
@@ -1,41 +1,88 @@
...
+[source,xml,diff-id=1,diff-type=noncompliant]
 ----
-// file: ejb-interceptors.xml
+<!-- ejb-interceptors.xml -->
+
 <assembly-descriptor>
- <interceptor-binding> <!-- should be declared in ejb-jar.xml -->
+ <interceptor-binding>
       <ejb-name>*</ejb-name>
       <interceptor-class>com.myco.ImportantInterceptor</interceptor-class> <!-- Noncompliant; will NOT be treated as default -->
```
Comment (gaetan-ferry-sonarsource) [inline rules/S3281/xml/rule.adoc:63]: I am not too sure what "Will not be treated as default" means here.

"by default"? "as a default interceptor"?

File: rules/S3281/xml/rule.adoc [hunk 2]
```diff
@@ -1,41 +1,88 @@
...
 </assembly-descriptor>
 ----
 
-
 == Resources
 
+=== Standards
+
 * https://owasp.org/Top10/A05_2021-Security_Misconfiguration/[OWASP Top 10 2021 Category A5] - Security Misconfiguration
 * https://owasp.org/www-project-top-ten/2017/A6_2017-Security_Misconfiguration[OWASP Top 10 2017 Category A6] - Security Misconfiguration
```
Comment (gaetan-ferry-sonarsource) [inline rules/S3281/xml/rule.adoc:87]:
```suggestion
* OWASP - https://owasp.org/Top10/A05_2021-Security_Misconfiguration/[Top 10 2021 Category A5 - Security Misconfiguration]
* OWASP - https://owasp.org/www-project-top-ten/2017/A6_2017-Security_Misconfiguration[Top 10 2017 Category A6 - Security Misconfiguration]
```

**Review by gaetan-ferry-sonarsource — CHANGES_REQUESTED** [review]
LGTM with some small improvement suggestions.

File: rules/S3281/xml/rule.adoc (hunk 1 above)
Comment (loris-s-sonarsource) [reply rules/S3281/xml/rule.adoc:65]: Good question, I kept it but wondered the same

Comment (loris-s-sonarsource) [reply rules/S3281/xml/rule.adoc:65]: I'm just gonna remove that part hahaha

**Review by gaetan-ferry-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**